ansible/files/admin_api_scripts/activate_read_replica.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+function read_replica_preparations {    
+    # Clear everything beforehand
+    if [[ -d /tmp/wal_fetch_dir ]]; then
+        rm -rf /tmp/wal_fetch_dir
+    fi
+    
+    mkdir /tmp/wal_fetch_dir
+    chown postgres:postgres /tmp/wal_fetch_dir
+    chmod 770 /tmp/wal_fetch_dir
+    
+    backup_name=$1
+    
+    # Stop database and empty it
+    systemctl stop postgresql
+    rm -rf /var/lib/postgresql/data/*
+
+    # Download base backup
+    wal-g backup-fetch /var/lib/postgresql/data "$backup_name" --config /etc/wal-g/config.json
+
+    # Signal for DB to be a Read Replica upon restart
+    touch /var/lib/postgresql/data/standby.signal
+
+    # Ensure that downloaded backup is owned by the postgres Linux user
+    find /var/lib/postgresql/data/ -exec chown postgres:postgres {} +
```

Comment:
can just be a `chown -R postgres:postgres /var/lib/pg/data`?

Reply:
touché